packages/module/src/catalog/QuickStartTile.tsx

```diff
@@ -40,6 +40,7 @@ const QuickStartTile: React.FC<QuickStartTileProps> = ({
     <FallbackImg
       className="co-catalog-item-icon__img--large"
       src={icon}
+      alt="Quick Start icon"
```

Comment:
Since this is decorative, it should be simply

```suggestion
      alt=""
```

Otherwise a screenreader will read "Quick Start icon" for each of these, which is not helpful to a visually impaired user.

Reply:
Sure Sam I'll update it. Thanks!